src/tools.h

```diff
@@ -25,6 +25,12 @@
 #include <string>
 #include <vector>
 
+typedef struct html_link
+{
+    std::string attribute;
+    std::string link;
+} html_link;
+
```

Comment:
I have to say that I would have naively use a `std::tuple` or a `std::pair`, but it is better with a specific structure.

Reply:
I was thinking about that as well but I wanted to be more expressive here to avoid confusion.